modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1137,28 +1135,26 @@ FIXME: Add docs.
 [[reconnect]]
 === Reconnect
 
-[%header,cols="20s,25a,30a,15a,10a"]
+[%header%autowidth.spread]
 |===
 | Field | Type | Description | Default Value | Required
-| Frequency a| Number | How often to reconnect (in milliseconds). | |
-| Count a| Number | The number of reconnection attempts to make. | |
-| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+| Frequency a| Number | How often in milliseconds to reconnect |  | 
+| Count a| Number | How many reconnection attempts to make. |  | 
 |===
 
 [[reconnect-forever]]
 === Reconnect Forever
 
-[%header,cols="20s,25a,30a,15a,10a"]
+[%header%autowidth.spread]
 |===
 | Field | Type | Description | Default Value | Required
-| Frequency a| Number | How often in milliseconds to reconnect. | |
-| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+| Frequency a| Number | How often in milliseconds to reconnect |  | 
```

Comment:
Needs a period. There may be some other occurrences of Frequency that need periods,